translation-web/src/main/java/eu/europeana/api/translation/config/TranslationConfig.java

```diff
@@ -16,10 +14,10 @@
  * Container for all settings that we load from the translation.properties file and optionally
  * override from translation.user.properties file
  */
-@Configuration
+@Configuration(BeanNames.BEAN_TRANSLATION_CONFIG)
 @PropertySources({@PropertySource("classpath:translation.properties"),
-    @PropertySource(value = "translation.user.properties", ignoreResourceNotFound = true)})
-public class TranslationConfig implements InitializingBean {
+@PropertySource(value = "translation.user.properties", ignoreResourceNotFound = true)})
```

Comment:
missing classpath - should be 'classpath:translation.user.properties'

Reply:
the translation.user.properties must not be in the classpath (to not include it in the docker image), they are provided as external file.  The integration test must be able to work without this file. The following command line arguments need to be used for running the app locally: --spring.config.name=translation,translation.user,application --spring.config.additional-location=optional:file:/opt/app/config/translation.user.properties